collectors/okta/okta_collector.js

```diff
@@ -28,6 +29,14 @@ const tsPaths = [
 
 class OktaCollector extends PawsCollector {
     
+    constructor(context, {aimsCreds, pawsCreds}){
+        super(context, 'paws',
+            AlAwsCollector.IngestTyps.LOGMSGS,
+            m_packageJson.version,
+            aimsCreds,
+            null, [healthChecks.oktaTokenHealthCheck], []);
+    }
```

Comment:
This is not the constructor pattern for the PAWS collector. I think we'll need to modify the actual constructor of the paws collector to accept an array of health checks.

https://github.com/alertlogic/paws-collector/blob/master/paws_collector.js#L73-L83